enrollment-server-onboarding-provider-zenid/pom.xml

```diff
@@ -77,6 +77,7 @@
                             <generateModels>true</generateModels>
                             <generateModelTests>false</generateModelTests>
                             <generateSupportingFiles>false</generateSupportingFiles>
+                            <skipValidateSpec>true</skipValidateSpec>
```

Comment:
The current ZenID swagger is not valid. I am waiting for information on whether we can get a better version. Otherwise, this option is required.

Reply:
We do not have a valid swagger yet. Let's merge it to make it work. We may improve later on.